lib/parallel_cucumber/main.rb

```diff
@@ -30,8 +30,12 @@ def run
         exit(1)
       end
 
-      all_tests = Helper::Cucumber.selected_tests(@options[:cucumber_options], @options[:cucumber_args])
-
+      begin
+        all_tests = Helper::Cucumber.selected_tests(@options[:cucumber_options], @options[:cucumber_args])
+      rescue StandardError => e
+        Hooks.fire_on_dry_run_error({ failure_reason: e.message, backtrace: e.backtrace, cucumber_options: @options[:cucumber_options], cucumber_args:  @options[:cucumber_args] })
```

Comment:
Please pass the whole error object to the block. otherwise we'll be missing `e.class` and `e.cause`

Reply:
also please could you omit brackets ? 

        Hooks.fire_on_dry_run_error(error: e, cucumber_options: @options[:cucumber_options], cucumber_args:  @options[:cucumber_args])

Reply:
not sure though how and why needed to use `cucumber_options` and `cucumber_args`

Reply:
Params are removed now, only error is passed